tiledb/cloud/rest_api/test/test_udf_api.py

```diff
@@ -28,6 +28,36 @@ def setUp(self):
     def tearDown(self):
         pass
 
+    def test_get_ud_fs(self):
```

Comment:
`ud_fs` -> `udfs`?

Reply:
This is from the autogenerated code, there is something funky in the parse of UDFs, it converts it to `udf_fs` instead of `udfs` or `u_d_f_s`.